Enterprise-CMCS/eAPD
# Tbolt/4559 update apd overview mmis read only

Resolves #4559 

### Description
Updates APD Overview section in the export/read-only view for MMIS APDs

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4601

### Significant changes or possible side effects
No side effects

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| a HITECH apd | no values are entered in APD Overview | renders correctly | jest |
| a HITECH apd | valid data is entered in APD Overview | renders correctly | jest |
| a MMIS apd | no values are entered in APD Overview | renders correctly | jest |
| a MMIS apd | valid data is entered in APD Overview | renders correctly | jest |

### Steps to manually verify this change

1. Login and view a HITECH apd
2. Verify no changes are present in APD Overview section of read-only/export view
3. Use chromatic to see the changes for MMIS
4. Verify design updates are present in APD Overview section of read-only/export view for MMIS

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
